src/cpp/multisolver_interface/COIN_common_functions.cpp

```diff
@@ -94,9 +94,16 @@ void fill_rhs_from_bounds(const double* rowLower,
         {
             if (rowUpper[i] < COIN_DBL_MAX)
             {
-                auto error = LOGLOCATION + "ERROR : Row " + std::to_string(i)
-                             + " has two different RHS, both right and left.";
-                throw GenericSolverException(error);
+                if (rowLower[i] == rowUpper[i])
```

Comment:
I do not really get when we can get into this `if` block without being passed into the one at line 89